tensorflow_federated/tools/python_package/build_python_package.sh

```diff
@@ -72,13 +72,13 @@ main() {
   python --version
   pip install --upgrade "pip"
   pip --version
+  # The `plat-name` manylinux tag should match GLIBC version returned by
+  # `ldd --version`.
   ldd --version
 
   # Build the Python package.
-  pip install --upgrade setuptools wheel
-  # The manylinux tag should match GLIBC version returned by `ldd --version`.
-  python "tensorflow_federated/tools/python_package/setup.py" bdist_wheel \
-      --plat-name=manylinux_2_31_x86_64
+  pip install --upgrade build
+  python -m build --wheel
```

Comment:
What platform is set for the package in this case?

Reply:
It's determined by the pyproject.toml file, in this case it's the same.